include/xsk/arc/assembler.hpp

```diff
@@ -25,7 +25,7 @@ class assembler
 
 public:
     assembler(context const* ctx);
-    auto assemble(assembly const& data) -> buffer;
+    auto assemble(assembly const& data, std::string const& string = "") -> buffer;
```

Comment:
Empty string should be initialize with empty braces rather than "" due to clang bug I reported.

https://github.com/llvm/llvm-project/issues/64535
https://github.com/llvm/llvm-project/issues/64536